s4e-web/src/app/views/map-view/state/product/product.service.ts

```diff
@@ -37,21 +50,46 @@ export class ProductService {
       .subscribe((data) => this.store.set(data));
   }
 
+  getLastAvailableScene() {
+    const product = this.query.getActive();
+
+    if (product == null) {
+      return;
+    }
+
+    this.store.ui.update(product.id, state => ({...state, isLoading: true}));
+    this.http.get<MostRecentScene>(`${environment.apiPrefixV1}/products/${product.id}/scenes/most-recent`,
+      {params: {timeZone: timezone()}}
+    ).pipe(
+      finalize(() => this.store.ui.update(product.id, state => ({...state, isLoading: false})))
```

Comment:
Why haven't you used handleHttpRequest$ operator? Why do you not handle errors in this pipe?